Gruntfile.js

```diff
@@ -35,7 +35,7 @@ module.exports = function( grunt ) {
       },
       dist: {
         files: {
-          "dist/vtt.min.js": distFiles
+          "dist/vtt.min.js": "dist/vtt.js"
```

Comment:
Hm... I think for now, I'd like to leave as is. Once we get around to updating from mozilla/vtt.js with 0.13, we can release a 1.0 with different dist name.